packages/cli/src/index.ts

```diff
@@ -390,41 +390,32 @@ const main = async () => {
       const fileType = lstatSync(targetPath).isDirectory()
         ? 'subdirectory'
         : 'file';
```

Comment:
Do we still support deploying a single file like this? I thought it was removed recently and you can only deploy a directory?

Reply:
Correct. There's probably a few places in the codebase that we can simplify now because of that. Perhaps not relevant for this PR though.